src/Compilers/VisualBasic/Portable/Lowering/Rewriter.vb

```diff
@@ -32,8 +36,14 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
             Dim rewrittenNodes As HashSet(Of BoundNode) = Nothing
             Dim flags = If(allowOmissionOfConditionalCalls, LocalRewriter.RewritingFlags.AllowOmissionOfConditionalCalls, LocalRewriter.RewritingFlags.Default)
             Dim localDiagnostics = DiagnosticBag.GetInstance()
+            dynamicAnalysisSpans = ImmutableArray(Of SourceSpan).Empty
 
             Try
+                Dim dynamicInstrumenter As DynamicAnalysisInjector =
+                    If(instrumentForDynamicAnalysis,
+                        DynamicAnalysisInjector.TryCreate(method, body, New SyntheticBoundNodeFactory(method, method, body.Syntax, compilationState, diagnostics), diagnostics, debugDocumentProvider, Instrumenter.NoOp),
```

Comment:
The current arrangement gives TryCreate the same signature for both VB and C#. The suggested change would result in different signatures (because in the C# implementation the method body factory already needs to exist outside the method). My preference is to leave the signature as is.